src/bindings/jna/libelektra/src/main/java/org/libelektra/Plugin.java

```diff
@@ -1,56 +1,83 @@
 package org.libelektra;
 
-import org.libelektra.exception.KDBException;
+import javax.annotation.Nonnull;
 
 /**
- * This is a Java representation of a plugin.
+ * Java representation of an Elektra plugin
  */
 public interface Plugin {
 
 	/**
-	 * Gets the config which was used to configure the plugin
-	 *
-	 * @return A KeySet containing the configuration of the plugin
+	 * Return value for plugin methods: An error occurred inside the plugin function
+	 */
+	static final int PLUGIN_STATUS_ERROR = -1;
```

Comment:
We are already in `Plugin`, do we need `PLUGIN_`? (also see others)

```suggestion
	static final int STATUS_ERROR = -1;
```

I am okay with it, though. (To have it similar to C.)

Reply:
renamed